packages/lumx-react/src/components/button/IconButton.test.tsx

```diff
@@ -79,7 +79,7 @@ describe(`<${IconButton.displayName}>`, () => {
             const { wrapper, buttonRoot } = setup(props);
             expect(wrapper).toMatchSnapshot();
 
-            expect(buttonRoot).toHaveClassName(props.className);
+            expect(buttonRoot).toHaveClassName(props.className as string);
```

Comment:
Why do you need to cast the className here?

Reply:
Because `HasClassName` has `className?: string`. The `className` is always optional (previously it had the implicit type `any` and thus the cast was not necessary)